pkg/apis/kops/validation/legacy.go

```diff
@@ -167,6 +167,24 @@ func ValidateCluster(c *kops.Cluster, strict bool) *field.Error {
 		}
 	}
 
+	// Check Canal Networking Spec if used
+	if c.Spec.Networking.Canal != nil {
+		action := c.Spec.Networking.Canal.DefaultEndpointToHostAction
+		if action != "" {
```

Comment:
we could probably shorten this with a case statement instead

```go
switch action {
case "", "ACCEPT", "DROP", "RETURN":
default: 
     return field.Invalid(fieldSpec.Child("Networking", "Canal", "DefaultEndpointToHostAction"), action, fmt.Sprintf("Unsupported value: %s, supports ACCEPT, DROP or RETURN", action))
}
```

Reply:
Yup good idea, will update.